### PR TITLE
Update how gov work, embassies, world location, and past foreign sec pages

### PR DESCRIPTION
**Pull request by injms**
This is the work towards moving pages that Whitehall renders towards reaching WCAG 2.1. The main aim is the new focus states that come from version 3 of GOV.UK Frontend. 

Where required, the markup has been made more accessible - **but this isn't a complete move towards using all of GOV.UK Frontend** - we are aware that plenty more improvements could be made.

🗃 [Trello card](https://trello.com/c/KJo6tZXo/58-5-update-whitehall-to-govukpublishingcomponents-v21x-with-compatibility-mode-on)

This pull request covers these pages:
 - [Get Involved](https://www.gov.uk/government/get-involved)
 - [Embassies](https://www.gov.uk/government/world/embassies)
 - [How government works](https://www.gov.uk/government/how-government-works)
 - [Past Foreign Secretaries](https://www.gov.uk/government/history/past-foreign-secretaries)

### [Get Involved](https://www.gov.uk/government/get-involved)
Open/Closed consultations large link before:
Screen Shot 2019-10-10 at 11 23 37

Open/Closed consultations large link after:
Screen Shot 2019-10-10 at 11 23 56

### [Embassies](https://www.gov.uk/government/world/embassies):

Before:
Screen Shot 2019-10-10 at 11 35 43

After:
Screen Shot 2019-10-10 at 11 37 07

### [How government works](https://www.gov.uk/government/how-government-works)

Minister circles before:
The difference is appearance is because of the difference in states and styles:
(focus state) (hover state) (normal state) (inverted style)

Screen Shot 2019-10-10 at 11 46 26

Minister circles after:
Screen Shot 2019-10-10 at 11 42 57
Screen Shot 2019-10-10 at 11 48 39

---

Government departments and agencies numbers before:
Screen Shot 2019-10-10 at 11 41 41

Government departments and agencies numbers after:
Screen Shot 2019-10-10 at 13 03 12

---
### [Past Foreign Secretaries](https://www.gov.uk/government/history/past-foreign-secretaries)

Notable Foreign Secretaries before:

Two focusable links present.
Screen Shot 2019-10-10 at 12 01 56
Screen Shot 2019-10-10 at 12 04 23

Notable Foreign Secretaries after:

Image link now `aria-hidden="true"` and `tabindex="-1"`.
Screen Shot 2019-10-10 at 12 03 58

### CODE REVIEW

**Comment by injms** [comment]
> @injms This commit [4ba6a17](https://github.com/alphagov/whitehall/commit/4ba6a17b8b4cd2a6f333d2ed60e4c459d6dfba0d) is quite big and difficult for me to grasp. I know that it's all for one template, but if it could be broken down for the smallest change for the page per commit that would really help.

Good point - and I've broken it down into more manageable chunks. Some commits, like ae9bab72cd6feffbdd9d321b9e9f7fe8a6e62e5b, might still look large - however most of them are just repetitive. (Fifty-four changed lines are down to extra `govuk-body` classes on paragraphs.)

Because I've gone back and select what to commit by highlighting it, some of the Sass in the commits might have unbalanced braces - but the final commit is fine.

File: app/views/home/how_government_works.html.erb [hunk 1]
```diff
@@ -3,18 +3,24 @@
...
+        } %>
+        <p class="govuk-body-l">In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out <a class="govuk-link" href="#who-runs-government">who runs government</a> and <a class="govuk-link" href="#how-government-is-run">how government is run</a>, as well as learning about the <a class="govuk-link" href="#history-uk-government">history of government</a>.</p>
+    </div>
     </div>
   </header>
 
   <div class="block">
     <div class="inner-block">
-      <h2 class="coalition" id="who-runs-government">Who runs government</h2>
+      <%= render "govuk_publishing_components/components/heading", {
```
Comment (chao-xian) [inline app/views/home/how_government_works.html.erb]: This was rendered with a bottom border. We should add an `<hr>` in to keep the design of the template consistent.

Also the styling of this h2 is exactly the same as the h3 below for "The Prime Minister"

File: app/views/home/how_government_works.html.erb [hunk 2]
```diff
@@ -50,15 +58,20 @@
         </div>
         <div class="one-half">
           <div class="inner-block">
-            <h3>The Cabinet</h3>
-            <p>The Cabinet is made up of the senior members of government. Every week during Parliament, members of the Cabinet (Secretaries of State from all departments and some other ministers) meet to discuss the most important issues for the government.</p>
-            <p><a href="/government/ministers">See who is in the Cabinet</a></p>
+            <%= render "govuk_publishing_components/components/heading", {
```
Comment (chao-xian) [inline app/views/home/how_government_works.html.erb]: Again this had a border top, so the template is no longer consistent.

File: app/views/home/how_government_works.html.erb [hunk 3]
```diff
@@ -50,15 +58,20 @@
...
+              heading_level: 3,
+              margin_bottom: 5,
+            } %>
           </div>
         </div>
       </div>
 
       <div class="full-width-section ministers">
-          <h3>Ministers</h3>
+        <%= render "govuk_publishing_components/components/heading", {
```
Comment (chao-xian) [inline app/views/home/how_government_works.html.erb]: Another border top needs recreating.

File: app/views/home/how_government_works.html.erb [hunk 4]
```diff
@@ -70,11 +83,19 @@
           <p><%= link_to "See full list of ministers", ministerial_roles_path %></p>
       </div>
 
-     <h2 id="how-government-is-run">How government is run</h2>
+      <%= render "govuk_publishing_components/components/heading", {
```
Comment (chao-xian) [inline app/views/home/how_government_works.html.erb]: Similar to above - no bottom border and the styling is the same as for the h3s.

File: app/views/home/how_government_works.html.erb [hunk 5]
```diff
@@ -317,8 +395,12 @@
         </div>
       </div>
 
-
-      <h2 id="history-uk-government">History of government</h2>
+      <%= render "govuk_publishing_components/components/heading", {
```
Comment (chao-xian) [inline app/views/home/how_government_works.html.erb]: Similar to above - no bottom border and the styling is the same as for the h3s.

File: app/views/home/how_government_works.html.erb [hunk 6]
```diff
@@ -234,54 +302,61 @@
...
           </div>
         </div>
       </div>
 
       <div class="thirds-width-section gov-types">
         <div class="one-third">
           <div class="inner-block">
-            <h3>Devolved government</h3>
-            <p>In <a href="http://www.gov.scot/" rel="external">Scotland</a>, <a href="http://gov.wales/" rel="external">Wales</a> and <a href="https://www.northernireland.gov.uk/" rel="external">Northern
+            <%= render "govuk_publishing_components/components/heading", {
```
Comment (chao-xian) [inline app/views/home/how_government_works.html.erb]: These are all also missing margin tops.

File: app/assets/stylesheets/frontend/views/_how-gov-works.scss [hunk 7]
```diff
@@ -275,14 +263,6 @@
   p.small-print {
     @include ig-core-14;
   }
-  h2.coalition {
-    border-color: black;
-  }
-  .coalition h3 {
```
Comment (chao-xian) [inline app/assets/stylesheets/frontend/views/_how-gov-works.scss]: I think this was needed to maintain the styles which I commented are missing in a previous commit on headings.

**Comment by chao-xian** [comment]
There are a couple of things that need addressing but also I think this PR is too large. I've split out the first 2 commits which look like they could be in a separate PR, for example https://github.com/alphagov/whitehall/pull/5076 . It'd be good to split this PR up further, as some of the other changes not related to the "How government works" page etc. could be reviewed and shipped separately/not be blocked by changes required for that. And review can be done not just by me as an added advantage.

**Comment by injms** [comment]
Closed as this requires a complete rework.

File: app/views/home/how_government_works.html.erb (hunk 1 above)
Comment (injms) [reply app/views/home/how_government_works.html.erb]: The styling is a limitation of the component. I'll undo all the components and revert most of the changes here.

**Comment by injms** [comment]
This work has been split up and placed in separate pull requests:

* #5076  
* #5081 
* #5082 
* #5083

**Pull request closed without merging**